api/scenario_api_utils/CMakeLists.txt

```diff
@@ -1,62 +1,28 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(scenario_api_utils)
 
-add_compile_options(-std=c++14)
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  roscpp
-  geometry_msgs
-  tf2_ros
-  tf2_geometry_msgs
-)
+find_package(ament_cmake_auto REQUIRED)
+ament_auto_find_build_dependencies()
 
 find_package(yaml-cpp REQUIRED)
 
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES scenario_api_utils
-  CATKIN_DEPENDS roscpp geometry_msgs tf2_ros tf2_geometry_msgs
-#  DEPENDS system_lib
-)
-
-include_directories(
-  include
-  ${catkin_INCLUDE_DIRS}
-)
-
-add_library(scenario_api_utils SHARED
+ament_auto_add_library(scenario_api_utils SHARED
   src/scenario_api_utils.cpp
 )
-
-add_dependencies(scenario_api_utils
-  ${${PROJECT_NAME}_EXPORTED_TARGETS}
-  ${catkin_EXPORTED_TARGETS})
-
 target_link_libraries(scenario_api_utils
-  ${catkin_LIBRARIES}
   ${YAML_CPP_LIBRARIES}
 )
 
-install(DIRECTORY include/
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
-)
-
-install(TARGETS scenario_api_utils
-  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION}
-)
-
-install(DIRECTORY include/${PROJECT_NAME}/
-  DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
-)
-
-catkin_install_python(
-  PROGRAMS scripts/ros_time_publisher.py
-  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+# Install python script  
+install(PROGRAMS scripts/ros_time_publisher.py
+  DESTINATION lib/${PROJECT_NAME}
```

Comment:
You did it right. The [migration guide](https://index.ros.org/doc/ros2/Contributing/Migration-Guide/) also recommends this mapping of  locations

    CATKIN_PACKAGE_BIN_DESTINATION: lib/${PROJECT_NAME}